reviews/migrations/0014_auto_20220824_1628.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.14 on 2022-08-24 14:28
+
+from django.db import migrations
+
+
+def choose_review_type(apps, schema_editor):
```

Comment:
This migration should work in theory, but I'm a bit afraid of the 'in practice' part. The DB is full of junk...

Not that you should change anything, I'm just voicing my concern and suggesting we should keep an eye on it when deploying it :P

Reply:
> This migration should work in theory

If I had the migrations the right way round, then maybe! The order has now been fixed, and `null` is allowed on `review_type` to span the intermediate deadzone.

As an aside, I have now learned that constants like `review.SUPERVISOR` are actually not available in historical models such as those you get in data migrations, unless they are contained in a Meta class. That's why there's a hardcoded 0 there now.